### PR TITLE
implement move speed icons

### PR DESCRIPTION
**Pull request by Ellamental2**
I deleted my fork forgetting this was open...

so i made it again and am including an option to check if the module is active before running the code.

### CODE REVIEW

**Comment by Ellamental2** [comment]
image

File: module/settings.js [hunk 1]
```diff
@@ -802,6 +802,15 @@ export function LoadSystemSettings() {
             category: "playtest",
             onChange: debouncedReload
         })
+
+        game.settings.register("ptu", "showMovementIcons", {
```
Comment (dylanpiera) [inline module/settings.js:806]: You do need to stop forgetting to include a category @Ellamental2

**Review by dylanpiera — CHANGES_REQUESTED** [review]
Add category and it can be merged (I can do this myself too after reviewing all your PRs)

**Comment by dylanpiera** [comment]
> Add category and it can be merged (I can do this myself too after reviewing all your PRs)

Ignore me it's client-side. Perfect.

**Pull request merged**